src/api/student.ts

```diff
@@ -11,6 +11,24 @@ import { getPlannerItemsMask, getPlannerItemsOAuth, UpcomingAssignment } from '.
 const BASE_URL: string = `${config.get('osuApi.baseUrl')}/students`;
 const router = Router();
 
+interface GradeTerm {
+  attributes: {
+    term: string;
+  }
+}
+/**
+ * This is a compare function based on the term attribute
+ * @param a Grade data provided by the api
+ * @param b Grade data provided by the api
+ * @returns [number] order in which a is compared to b
+ */
+const sortGradesByTerm = (a: GradeTerm, b: GradeTerm): number => {
+  if (!b) return 0;
+  if (a.attributes.term < b.attributes.term) return 1;
+  if (b.attributes.term > a.attributes.term) return -1;
```

Comment:
These two if's are the same logic, it will never return a -1. I have a rewrite running but Mauricio and I want to try and get it to fail a test